.github/workflows/mingw.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         uses: actions/cache@v1.1.2
         with:
           path: D:\a\_temp\msys
-          key: windows-mingw
+          key: windows-mingw-2
 
```

Comment:
After remade from master this shouldn't be required, since it have retry in case of cache doesn't match

Cc: @ygj6

Reply:
@okhowang can you remove cahce key changes please?

Reply:
removed